frontend/src/modules/layout/pages/paywall-page.vue

```diff
@@ -88,6 +105,9 @@ const props = defineProps({
 })
 
 const page = computed(() => pageContent[props.module])
+const computedFeaturePlan = computed(() => {
```

Comment:
On this file (line 18) there is also a pre title with "Growth feature". Could you also update that one? 🙏